src/service/service.toit

```diff
@@ -145,6 +148,9 @@ class ArtemisServiceProvider extends ChannelServiceProvider
   device-id -> ByteArray:
     return device_.id.to-byte-array
 
+  reboot --safe-mode/bool -> none:
+    // TODO(florian): implement this.
```

Comment:
Maybe throw for now?

Or go to the STOP runlevel and add a TODO for the safe mode.

Reply:
going with "throw" for now.

Reply:
Fixed.